docs/001_develop/03_client-capabilities/006_charts/001_g2plot-chart/01_g2plot-chart.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: 'G2Plot Chart'
+sidebar_label: 'Overview'
+id: g2plot-chart
+keywords: [genesis, foundation, ui, chart, g2plot-chart, g2plot]
+tags:
+  - connected
+  - datasource
+  - chart
+  - g2plot-chart
+  - g2plot
+---
+
+# G2Plot Chart
+
```

Comment:
I think we need a more friendly introduction around this. We have more than this here today https://learn.genesis.global/docs/web/web-components/presentation/chart/

Reply:
@jay-taylerson this will be done in a separate PR

this is just to add API docs + required setup/configs for that